### ISSUE TITLE
WCAG; request signup flow page title should be heading element

### ISSUE THREAD

**Issue by jamalv** [teamforus/forus#2182]
In step 1 of this process, the heading "City pass application" is not formatted as a heading. Use h1 element for this text. All other steps use headings and subheadings that are not formatted as headers.

When this header is formatted with a header element, the structural information that the text provides is also transferred to auxiliary software.

Convert this header to a heading element.

**Comment by SjoerdHilhorst** [comment]
gefixt
